templates/catalog/_partials/miniatures/product.tpl

```diff
@@ -168,14 +168,14 @@
               {/block}
             </div>
 
-            {if $product.add_to_cart_url}
+            {if $product.add_to_cart_url && !$product.is_customizable && !$product.attributes|@count}
```

Comment:
```suggestion
            {if !$configuration.is_catalog && $product.add_to_cart_url && !$product.is_customizable && !$product.attributes|@count}
```

Reply:
checking for this is already inside add_to_cart_url

Reply:
then maybe other conditions should be there too to avoid so many conditions in the template

Reply:
Is this config still used in the template?
![obraz](https://user-images.githubusercontent.com/5007456/210338929-2b806e2d-06ce-4356-b70b-684cb0ba8fa1.png)
![obraz](https://user-images.githubusercontent.com/5007456/210339042-b2a3a46a-7cca-4549-8611-1c26691ab9bf.png)

Reply:
@SharakPL what's the variable exposed with that option? I can't manage to find it by myself

Reply:
@NeOMakinG I don't think it is available, I think we should add it inside `FrontController::getTemplateVarConfiguration()`

Reply:
Actually I think this is already handled too. Name is `allow_add_variant_to_cart_from_listing`

https://github.com/PrestaShop/PrestaShop/blob/c15bdae80e963714c55c4438aeb5a5adaec1e2d7/src/Adapter/Presenter/Product/ProductListingLazyArray.php#L38-L49

Reply:
Well it's probably doing the tricks, seems simpler now